build/build.mk

```diff
@@ -140,6 +140,12 @@ linux/harvest-tests: GOARCH=amd64
 linux/harvest-tests: deps
 	go test ./test/harvest -tags="harvest" -v
 
+.PHONY: macos/harvest-tests
```

Comment:
do we still need this?

Reply:
We are not running this on CI *for now* but we will.
OTOH you can run this locally.